Sources/SparkConnect/DataFrame.swift

```diff
@@ -342,6 +342,21 @@ public actor DataFrame: Sendable {
     return self
   }
 
+  var storageLevel: StorageLevel {
```

Comment:
I wonder why this is `var` instead of a function? E.g., in `Dataset.scala`, it is `def storageLevel: StorageLevel`.